assets/js/src/form_editor/store/reducers/history_record.ts

```diff
@@ -0,0 +1,83 @@
+import { isEqual } from 'lodash';
+
+export const createHistoryRecord = (state) => {
```

Comment:
I think this whole file deserves a test, what do you think?

Reply:
👍🏻 Absolutely yes.